lambdaFiles/mainLambda.js

```diff
@@ -79,11 +78,47 @@ function resendText(event, callback) {
   callback(undefined, responseSuccess);
 }
 
+// updates the checkin message (kekw yes or no buttons) with a message that thanks them for their response
+// prevents spamming of kekw buttons bc buttons no longer accessible
+function updateCheckinMessage(responseUrl) {
+  var postData = JSON.stringify({
+    replace_original: "true", // edits original message
+    text: "Thanks for your response!", // TODO: make this more specific- who responded?  did you meet?
+  });
+
+  const searchTerm = ".com/";
+  const indexOfFirst = responseUrl.indexOf(searchTerm);
+
+  const pathSubstring = responseUrl.substring(indexOfFirst + 4);
+
+  var options = {
+    hostname: "hooks.slack.com",
+    path: pathSubstring,
+    method: "POST",
+  };
+
+  var req = https.request(options, (res) => {
+    console.log("statusCode:", res.statusCode);
+
+    res.on("data", (d) => {
+      process.stdout.write(d);
+    });
+  });
+
+  req.on("error", (e) => {
+    console.error(e);
+  });
+
+  req.write(postData);
+  req.end();
+}
```

Comment:
this post request is quite ugly bc i can't use the `requests` or `axios` library ;-; 
perhaps we should invest time into trying to import npm modules for lambdas? (or maybe not bc we're almost done xd)
https://stackoverflow.com/questions/34437900/how-to-load-npm-modules-in-aws-lambda

Reply:
I think it's fine, the code is still understandable.